csi/controller_server.go

```diff
@@ -732,23 +732,49 @@ func (cs *ControllerServer) ControllerExpandVolume(ctx context.Context, req *csi
 	if existVol.State != string(types.VolumeStateDetached) {
 		return nil, status.Errorf(codes.FailedPrecondition, "Invalid volume state %v for expansion", existVol.State)
 	}
-	volumeSize, err := strconv.ParseInt(existVol.Size, 10, 64)
+	existingSize, err := strconv.ParseInt(existVol.Size, 10, 64)
```

Comment:
The naming is a little bit weird... How about `currentSize`?

Reply:
It is `existingSize` because it doesn't get updated and is used below in the log message.
Since we retrieve an updated volume below, therefore I find keeping existing Size better.